.gitignore

```diff
@@ -1,29 +1,2 @@
-# eclipse conf file
-.settings
-.classpath
-.project
-.manager
-.scala_dependencies
-.cache
-
-# idea conf files
-.idea
-*.iml
-*.ipr
-*.iws
-
-# building
-target
-build
-null
-tmp
-temp
-dist
-test-output
-build.log
-
-# misc
-.DS_Store
-
-# project files to ignore
-!/results/.gitkeep
+/target
+/.mvn/wrapper/*.jar

```

Comment:
We went for the simplest gitignore (only project-specific files) in the other demo projects but not this one... I assumed the minimalist gitignore makes sense, but either way we should be consistent between the different demo projects.